cmd/kubemark/hollow-node.go

```diff
@@ -51,14 +51,16 @@ import (
 )
 
 type HollowNodeConfig struct {
-	KubeconfigPath      string
-	KubeletPort         int
-	KubeletReadOnlyPort int
-	Morph               string
-	NodeName            string
-	ServerPort          int
-	ContentType         string
-	UseRealProxier      bool
+	KubeconfigPath       string
```

Comment:
Btw - since you're touching this file anyway, could you add a TODO above that we should eventually separate this into kubelet and kube-proxy (instead of mixing them up)? The issue for that is - https://github.com/kubernetes/kubernetes/issues/45650

Reply:
Done